_data/simple-icons.json

```diff
@@ -6127,6 +6127,11 @@
             "hex": "EA4335",
             "source": "https://fonts.gstatic.com/s/i/productlogos/gmail_2020q4/v8/192px.svg"
         },
+        {
+            "title": "GMC",
+            "hex": "E21836",
+            "source": "https://www.gmc.com/content/dam/gmc/na/us/english/primary-nav-icons/GMC-153x28.svg"
```

Comment:
As per https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#source-guidelines we should avoid linking directly to the SVG files themselves and rather link to the page that has the SVG displayed on it.